corehq/apps/tour/tours.py

```diff
@@ -0,0 +1,33 @@
+from corehq.apps.tour.models import StaticGuidedTour, GuidedTourStep
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_noop as _
+
+
+SIMPLE_NEW_APP = StaticGuidedTour('simple_new_app', [
```

Comment:
Recommendation: separate the stateful components (cache) from the static/constant components. The API might look something like this:

```py
tours.get(tours.SIMPLE_NEW_APP).setup(...)
```